crates/pallet-subspace/src/lib.rs

```diff
@@ -596,7 +598,7 @@ pub mod pallet {
 
         /// Enable storage access for all users.
         #[pallet::call_index(5)]
-        #[pallet::weight(<T as Config>::WeightInfo::enable_authoring_by_anyone())]
+        #[pallet::weight(< T as Config >::WeightInfo::enable_authoring_by_anyone())]
```

Comment:
are whitespaces in this file expected or unintentional ?

Reply:
This is what `rustfmt` does for me. Surely not intentional and I'm not happy with what it does actually. Does it not format the same way for you?